src/js/views/scalebar_form_view.js

```diff
@@ -0,0 +1,208 @@
+
+window.UNIT_SYMBOLS = {"PICOMETER": {symbol: "pm", microns: 0.000001},
```

Comment:
Value are retrieved in the script but not in the js.
I still think this should come from the server.
this will require a call to get the value and "populate" the UI

Reply:
I will still have to hard-code the units that I want available here as I don't want the UI to offer all of them: ```['ANGSTROM', 'ASTRONOMICALUNIT', 'ATTOMETER', 'CENTIMETER', 'DECAMETER', 'DECIMETER', 'EXAMETER', 'FEMTOMETER', 'FOOT', 'GIGAMETER', 'HECTOMETER', 'INCH', 'KILOMETER', 'LIGHTYEAR', 'LINE', 'MEGAMETER', 'METER', 'MICROMETER', 'MILE', 'MILLIMETER', 'NANOMETER', 'PARSEC', 'PETAMETER', 'PICOMETER', 'PIXEL', 'POINT', 'REFERENCEFRAME', 'TERAMETER', 'THOU', 'YARD', 'YOCTOMETER', 'YOTTAMETER', 'ZEPTOMETER', 'ZETTAMETER']```.

Reply:
We need to find a better strategy in that case e.g. we add methods full, common metrics etc.
as mentioned on previous PR we really to start thinking of the usage of the API instead of "patching" as we do across the code base.

Reply:
See 25f4bc1